src/components/DocLink.tsx

```diff
@@ -1,94 +1,66 @@
 import React from "react"
 import { useTheme } from "@emotion/react"
-import styled from "@emotion/styled"
-import Icon from "./Icon"
+import { Icon, Box, Container, Text } from "@chakra-ui/react"
+import { AiOutlineArrowRight } from "react-icons/ai"
 import Link from "./Link"
-import Emoji from "./OldEmoji"
+import Emoji from "./Emoji"
 
 export interface IProps {
   children?: React.ReactNode
   to: string
-  className?: string
   isExternal?: boolean
 }
 
-const Container = styled(Link)<{ isExternal: boolean }>`
-  position: relative;
-  z-index: 1;
-  text-decoration: none;
-  display: flex;
-  flex-direction: row;
-  flex: 1;
-  width: 100%;
-  justify-content: space-between;
-  padding: 1rem;
-  border-radius: 2px;
-  color: ${({ theme }) => theme.colors.text};
-  border: 1px solid ${({ theme }) => theme.colors.border};
-  &:hover {
-    text-decoration: none;
-    box-shadow: 0 0 1px ${({ theme }) => theme.colors.primary};
-    background: ${({ theme }) => theme.colors.tableBackgroundHover};
-    border-radius: 4px;
-
-    svg {
-      fill: ${(props) => props.theme.colors.primary};
-      transition: transform 0.1s;
-      transform: scale(1.2)
-        rotate(${({ isExternal }) => (isExternal ? "-45deg" : "0")});
-    }
-  }
-  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
```

Comment:
We are not declaring this media query. We could use https://chakra-ui.com/docs/hooks/use-media-query for this.

Reply:
Hey @pettinarip, can you take a look at my recent modifications to the pull request. Specifically the modifications I made in my latest change for the useMediaQuery. I'm not sure whether I did it the right way.